sound/soc/sof/sof-audio.c

```diff
@@ -485,7 +485,7 @@ sof_prepare_widgets_in_path(struct snd_sof_dev *sdev, struct snd_soc_dapm_widget
 			if (ret < 0) {
```

Comment:
@ujfalusi The commit summary very Yoda like is! ("Do not unprepared used swidget instances on prepare ... set up")

Reply:
Unprepared used swidget instances on prepare, do not.  Herh herh herh.